ansible-ipi-install/README.md

```diff
@@ -418,6 +419,53 @@ With the `playbook.yml` set and in-place, run the `playbook.yml`
 $ ansible-playbook -i inventory/hosts playbook.yml
 ~~~
 
+## Disconnected Registry
+A disconnected registry can be used to deploy the cluster.
+This registry can exist or can be created.
+
+To use a disconnected registry, set the registries host name in 
+the `[registry_host]` group in the inventory file.
+
+### Using an Existing Disconnected Registry
+The `disconnected_registry_auths_file` and the 
+`disconnected_registry_mirrors_file` variables must be set.
+
+The `disconnected_registry_auths_file` variable should be json data. This will be appended to the `auths` section of the pull secret.
+
+The `disconnected_registry_mirrors_file` variable should contain the `additionalTrustBundle` and `imageContentSources` for the disconnected registry.
+~~~
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  The servers certificate should go here
+  -----END CERTIFICATE-----
+  
+imageContentSources:
+- mirrors:
+  - disconnected.example.com:5000/ocp4/openshift4
+  source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+- mirrors:
+  - disconnected.example.com:5000/ocp4/openshift4
+  source: registry.svc.ci.openshift.org/ocp/release
+~~~
+
+### Creating a New Disconnected Registry
+To create a new disconnected registry, the `disconnected_registry_auths_file` 
+and `disconnected_registry_mirrors_file` variables must not be set.
+
+The certificate information used to generate the host certificate must be defined.
+These variables must be defined as variables to the `registry_host` group in 
+the inventory file.
+~~~
+[registry_host:vars]
+cert_country=MyCountry
+cert_state=MyState
+cert_locality=MyCity
+cert_organization=MyCompany
+cert_organizational_unit=MyDepartment
```

Comment:
see above comment

Reply:
would this be confusing to put it there before discussing it?

Reply:
done in next commit